Client.Core/Flux/Internal/FluxCsvParser.cs

```diff
@@ -389,6 +396,16 @@ private void AddColumnNamesAndTags(FluxTable table, CsvReader columnNames)
                 var fluxColumn = GetFluxColumn(ii, table);
                 fluxColumn.Label = columnNames[ii + 1];
             }
+
+            var duplicates = table.Columns.GroupBy(col => col.Label)
+                .Where(rec => rec.Count() > 1)
+                .Select(label => label.Key).ToList();
+            if (duplicates.Any())
+            {
+                Console.WriteLine(
+                    $"The response contains columns with duplicated names: {string.Join(", ", duplicates)}\n" +
+                    "You should use the 'record.row' to access your data instead of 'record.values' dictionary.");
```

Comment:
Please use Camel case for names of fields:

```suggestion
                    "You should use the 'record.Row' to access your data instead of 'record.Values' dictionary.");
```